Userland/Services/Taskbar/TaskbarWindow.h

```diff
@@ -11,16 +11,19 @@
 #include <LibDesktop/AppFile.h>
 #include <LibGUI/Widget.h>
 #include <LibGUI/Window.h>
+#include <LibGfx/Alignment.h>
 #include <LibGfx/ShareableBitmap.h>
 #include <WindowServer/ScreenLayout.h>
 
-class TaskbarWindow final : public GUI::Window {
+class TaskbarWindow final : public GUI::Window
+    , public Config::Listener {
     C_OBJECT(TaskbarWindow);
 
 public:
     virtual ~TaskbarWindow() override;
 
     static int taskbar_height() { return 27; }
+    static int taskbar_width() { return 40; }
```

Comment:
This width causes the ... to appear after window names. We should either give a width such that `TextLayout` just gives up and doesn't do anything at all, or actually not render window titles when in vertical orientation.